docs/development/test-vectors.rst

```diff
@@ -860,6 +863,8 @@ Custom PKCS7 Test Vectors
 * ``pkcs7/amazon-roots.der`` - A DER encoded PCKS7 file containing Amazon Root
   CA 2 and 3 generated by OpenSSL.
 * ``pkcs7/enveloped.pem`` - A PEM encoded PKCS7 file with enveloped data.
+* ``pkcs7/enveloped-aes-256-cbc.pem`` - A PEM encoded PKCS7 file with 
```

Comment:
Is this encrypted under the public key of `/pkcs7/ca/ca.pem`? If so let's say that.

Reply:
It is actually encrypted under the public key of `x509/custom/ca/rsa_ca.pem`, which is used in most tests for the PKCS7 module. Adding this right now to the documentation.